src/graphs/Breadth_First_Search.re

```diff
@@ -0,0 +1,65 @@
+type node = {
+    id: string,
+    neighbours: list(node),
+};
+
+type pair = (string, list(string));
+
+let approx_num_nodes = 5;
```

Comment:
why not calculating the number of nodes when invoking BFS on a graph? `List.length`